src/components/Preview.js

```diff
@@ -24,22 +25,22 @@ class Preview extends React.Component {
                         </div>
                         <ul className="list__rs">
                             <li className="reset__list preview__email hidden">
-                                <a className="rs__element rs__email" href="mailto:">
+                                <a className="rs__element rs__email" href={`mailto:${email}`}>
                                     <i className="icon far fa-envelope"></i>
                                 </a>
                             </li>
                             <li className="reset__list preview__tel hidden">
-                                <a className="rs__element rs__tel" href="tel:">
+                                <a className="rs__element rs__tel" href={`tel:${tel}`}>
                                     <i className="icon fas fa-mobile-alt"></i>
                                 </a>
                             </li>
                             <li className="reset__list preview__linkedin hidden">
-                                <a className="rs__element link__linkedin" href="linkedin_icon" target="_blank" rel="noopener noreferrer">
+                                <a className="rs__element link__linkedin" href={`https://www.linkedin.com/in/${linkedin}`} target="_blank" rel="noopener noreferrer">
```

Comment:
There is a warning in the console that the rel and target are blank here - we should change this.